docker/Dockerfile.prod

```
@@ -1,4 +1,4 @@
-FROM node:22-alpine
+ROM node:22-alpine
```

Hadolint (2.12.0)

[error] unexpected 'R'
expecting '#', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, end of input, or whitespaces

(DL1000)